devnets/1620/shared/genesis.json

```diff
@@ -0,0 +1,842 @@
+{
+  "config": {
+    "chainId": 1620,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "berlinBlock": 0,
+    "londonBlock": 0,
+    "mergeForkBlock": 0,
+    "terminalTotalDifficulty": 0,
+    "shanghaiTime": 1683680926
```

Comment:
should be 1683917326 to match with my previous comment

Reply:
Updated.